src/Mastering.ts

```diff
@@ -22,14 +22,21 @@ export class MasteringClass {
   /**
    * Get mastering by scriptId & parameters
    * @param scriptId
-   * @param parameters
+   * @param parameters Object containing the audience item you want to retrieve.
+   * @param _public To store the mastered file in a public s3 folder. Default value is `false`. Warning - This will cause your mastered files to be public to anyone in the internet. Use this at your own risk.
+   * @param vast To create a VAST file of your mastered file. The `vast` flag only works if `public` is `True`.
    */
-  public retrieve(scriptId: string, parameters: PersonalisationParameters = {}): Promise<unknown> {
+  public retrieve(
+    scriptId: string,
+    parameters: PersonalisationParameters = {},
+    _public = false,
```

Comment:
`_public` or `public` ? I guess public is a reserved word right? 😅

Reply:
hahaha yes it is :) but since this is not in an object (dict), it wont matter to users. they will just put true or false and docs will show correctly